src/main/java/io/debezium/connector/spanner/db/dao/SchemaDao.java

```diff
@@ -35,6 +35,10 @@ public SpannerSchema getSchema(Timestamp timestamp) {
         return getSchema(timestamp, null);
     }
 
+    public DatabaseClient getDatabaseClient() {
```

Comment:
Why do we need to retrieve the database client here?

Reply:
![image](https://user-images.githubusercontent.com/43364656/234935069-68165ee5-fae6-41bf-a047-0954f5d4b3c5.png) Because we need to get the dialect of current database when adding the columns right here.

Reply:
Can we simply make this function public and use this function instead? https://github.com/debezium/debezium-connector-spanner/blob/af6f4cf4681d74fb01b67a63c86d48cd9437b4c8/src/main/java/io/debezium/connector/spanner/db/dao/SchemaDao.java#L205

Reply:
Done.